pages/common/dot.md

```diff
@@ -0,0 +1,11 @@
+# dot
+
+> A command-line tool to produce layered drawings of directed graphs.
+
+- Use output filenames based on input filename and format:
```

Comment:
From reading this, I don't quite understand what this command does. Perhaps it could be clarified a bit?

Reply:
How about this?

```markdown
- Create an image file and determine output filename based on input filename and selected format:

`dot -T{{format}} -O {{path/to/file.dot}}``
```

Reply:
Yeah, that should do nicely! Perhaps change `Create` to `Render` though.